mayhem/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build Stage
```

Comment:
I recommend removing the commented out parts of the Dockerfile.

Reply:
Done. Regarding the .o and .eo files, I believe the repo maintainer added those, not sure what they're for.

Reply:
I've cleaned up the .o and .eo files from the repo. They're the product of build the binary used in the mayhemfile.